@xen-orchestra/lite/src/stories/web-core/top-bottom-table/ui-top-bottom-table.story.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <ComponentStory
+    v-slot="{ properties }"
+    :params="[
+      prop('item-count').num(),
+      prop('total-item-count').num(),
```

Comment:
The `required` information is missing:
```suggestion
      prop('item-count').required().num(),
      prop('total-item-count').required().num(),
```
And remember to match the prop names with the actual ones used in the component.